### ISSUE TITLE
Importing from Roboform HTML

### ISSUE THREAD

**Issue by anantharam** [bitwarden/web#157]
I followed the steps to import my data from roboform and it seems bitwarden is not recognizing some of the logins, it recognizes some other logins but a few of logins failed to be imported.

An example login that failed is like below which contains the login details for the website https://0x00sec.org/, same issue is applicable to other websites like https://www.airbnb.co.in/, e.t.c.  : 

```
<DIV class="floatdiv orph">
<TABLE width="100%" class=nobr>
<TBODY>
<TR class=nobr>
<TD style="WIDTH: 1px"><IMG class=passcardicon src="RoboformSnapshot_files/0x00sec.bmp"></TD>
<TD style="WORD-BREAK: break-all"><SPAN class=caption style="VERTICAL-ALIGN: middle; WORD-BREAK: break-all">0x00sec</SPAN></TD></TR>
<TR>
<TD></TD>
<TD>
<TABLE width="100%">
<TBODY>
<TR align=left width="100%">
<TD class=subcaption style="WORD-BREAK: break-all; COLOR: gray" colSpan=3>0x0&shy;0sec.org</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>email:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top>abc&shy;defghi&shy;jkl&shy;r@gmail.com</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>use&shy;rname:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top>abc&shy;xyz</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>-no&shy;-name-:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top>ABC XYZ</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>-no&shy;-name-:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top>****</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>-no&shy;-name-:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top>5e7&shy;d0d63aa67446</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>-no&shy;-name-:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top>c2e&shy;eb6fe4c9&shy;3ae78ce0&shy;bedaa94a&shy;07a98</TD></TR>
<TR width="100%">
<TD width="40%" align=left class=field vAlign=top>use&shy;rname:</TD>
<TD></TD>
<TD width="50%" align=left class=field vAlign=top></TD></TR></TBODY></TABLE></TD></TR></TBODY></TABLE></DIV>
```

**Steps to reproduce :**

1. Create an account in https://0x00sec.org/ or https://www.airbnb.co.in/
1. Store the login credentials from that site in roboform
1. Export the login details from roboform
1. Import the html file from bitwarden
1. Go to the site and try to log in using bitwarden
1. Notice that the imported login detail only has site name and doesn't contain username and password

**Expected Result**

1. The user name and password fields should be read correctly from all websites during import from roboform.

**Comment by kspearrin** [comment]
Fixed in v2.0. We now support RoboForm CSV imports, which work much better.

**Comment by budsglobal** [comment]
Please fix the broken ROBOFORM .... I can NOT import from bitwarden csv ,  receive errors?????.
